.github/workflows/release.yml

```diff
@@ -0,0 +1,47 @@
+name: Release
+
+on:
+  push:
+    tags: [tst-v*, v*]
```

Comment:
Please only `tst-v*` for now, I don't want this flow to interfere (or break) the real release flow. We'll cut this over when we have confirmed that this is working and ready for prime time.